tasks.py

```diff
@@ -777,7 +838,6 @@ def unittest(
         command += " --verbosity 2"
 
     run_command(context, command)
-    run_command(context, "coverage lcov --include 'nautobot_design_builder/*' -o lcov.info")
```

Comment:
This will be coming back in cookiecutter-nautobot-app 2.5.0, but not part of the unitest, it will have it's own step.